### PR TITLE
[ᚬmaster] feat(keypair): add another generate keypair arg

### PR DESCRIPTION
**Pull request by KaoImin**
**What type of PR is this?**
feature

**What this PR does / why we need it**:
Add another method to generate key pairs. Now the tool can generate keypairs from a given vector of secp256k1 private keys. If the given keys number is larger than the value of argument `-n`, the program will panic.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by KaoImin** [comment]
e.g. 
image

File: devtools/keypair/src/main.rs [hunk 1]
```diff
@@ -33,7 +35,17 @@ struct Output {
 pub fn main() {
     let yml = load_yaml!("keypair.yml");
     let m = App::from(yml).get_matches();
-    let number = value_t!(m, "number", u32).unwrap();
+    let number = value_t!(m, "number", usize).unwrap();
+    let priv_keys = if let Ok(tmp) = values_t!(m.values_of("private_keys"), String) {
```
Comment (zeroqn) [inline devtools/keypair/src/main.rs]:
```suggestion
let priv_keys = values_t!(m.values_of("private_keys"), Sting).unwrap_or_default();
```

**Comment by huwenchao** [comment]
Too many duplicate codes. Try this: https://github.com/KaoImin/muta/pull/1/files

File: devtools/keypair/Cargo.toml [hunk 2]
```diff
@@ -1,19 +1,23 @@
 [package]
-name = "keypair"
+name = "muta-keypair"
 version = "0.1.0-alpha.0"
-authors = ["zero.qn <zero.qn@outlook.com>"]
+authors = ["Eason Gao <kaoimin@qq.com>"]
```
Comment (yejiayu) [inline devtools/keypair/Cargo.toml]: authors = ["Muta Dev <muta@nervos.org>"]

**Pull request merged**